README.md

```diff
@@ -2,14 +2,75 @@
 
 Experimental build script for custom rkt stage1-kvm images.
 
+stage1-builder relies on [ACI dependencies](https://github.com/appc/spec/blob/master/spec/aci.md).
+This allows us to base our images on the upstream stage1-kvm and only
+update the kernel.
+
+Note: at the time of writing, you have to pre-fetch all stage1 dependencies
+manually due to https://github.com/coreos/rkt/issues/2241
```

Comment:
I would move this note below, just before the other "rkt fetch" (line 43). Also please say which command line to use to pre-fetch the dependencies.